ai-library/base/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- crd.yaml
+- service-account.yaml
+- role.yaml
+- role-binding.yaml
+- operator.yaml
+namespace: kubeflow
```

Comment:
I am planning on using namespace `opendatahub` for ODH components, what do you think?

Reply:
That's probably a fine idea